### ISSUE TITLE
Double-clicking on a Chinese word will highlight just that word.

### ISSUE THREAD

**Issue by xfq** [w3c/character_phrase_tests#31]
[test](https://w3c.github.io/i18n-tests/exploratory/vanilla/?text=%E5%9B%BD%E5%AE%B6%E7%94%B1%E9%A2%86%E5%9C%9F%E3%80%81%E4%BA%BA%E6%B0%91%E3%80%81%E6%96%87%E5%8C%96%E5%92%8C%E6%94%BF%E5%BA%9C%E5%9B%9B%E4%B8%AA%E8%A6%81%E7%B4%A0%E7%BB%84%E6%88%90%E3%80%82&lang=zh-Hans&fontSize=60&width=730&height=216)

Double-click on 国家, 领土, 人民, 文化, 政府, 要素 and 组成. Test passes if two Chinese characters are selected in each case.

- Gecko: ✘ Firefox/75.0 macOS 10.15.4 Highlights the whole sequence of adjacent Chinese characters, with no word segmentation.
- Blink: ✓ Chrome/81.0.4044.138 macOS 10.15.4
- Webkit: ✓ Safari/13.1 macOS 10.15.4

**Comment by xfq** [comment]
@r12a I just added a test for Chinese. How should I add the test results (if it's OK)? It doesn't seem to be documented in https://github.com/w3c/i18n-tests/wiki/Using-the-interactive-exploratory-tests

**Comment by r12a** [comment]
Yes, i expect staff to do that, so that we can check the tests first.

**Comment by r12a** [comment]
See https://w3c.github.io/i18n-tests/results/int-grapheme-word-segmentation.html#chinese

**Comment by r12a** [comment]
Given that Blink & Safari tended to select 2 characters at a time for Japanese, it may be good to modify the test slightly to include a 1-letter and 3-letter word in Chinese, if such a thing makes sense.

**Comment by xfq** [comment]
> See https://w3c.github.io/i18n-tests/results/int-grapheme-word-segmentation.html#chinese

Ah, thanks. I added it in the first comment.

**Comment by xfq** [comment]
> Given that Blink & Safari tended to select 2 characters at a time for Japanese, it may be good to modify the test slightly to include a 1-letter and 3-letter word in Chinese, if such a thing makes sense.

I have updated the test and result. When double-clicking on 哭了, Blink and WebKit highlights 哭了 instead of just 哭.

-----

Background: 哭 is a verb (which means "to cry") and 了 is a perfective aspect particle used after verbs to indicate a completed action. (It is kind of like the case in https://github.com/w3c/character_phrase_tests/issues/30#issuecomment-630851379 but not exactly the same.)

**Comment by r12a** [comment]
I wonder whether that's a bug or a feature?

**Comment by xfq** [comment]
I think this is a bug, because they're two different words, but I'll raise this as a question in the clreq issue list.
